iothub/service/src/Amqp/AmqpCbsSessionHandler.cs

```diff
@@ -12,7 +12,9 @@ namespace Microsoft.Azure.Devices.Amqp
     /// Handles a single CBS session (for SAS token renewal) including the inital authentication and scheduling all subsequent
     /// authentication attempts.
     /// </summary>
+#pragma warning disable CA1852 // used in debug for unit test mocking
```

Comment:
We need to remember to do this when we add unit tests that access internals, otherwise it'll fail the release build due to static analysis.